src/game.cpp

```diff
@@ -1225,14 +1225,18 @@ bool game::cleanup_at_end()
     return true;
 }
 
-static int veh_lumi(vehicle *veh)
+static int veh_lumi( vehicle &veh )
```

Comment:
Yes, but that requires defining a `const` variant of `vehicle::lights`.

What does that code section actually do - its repeated in `lightmap.cpp` and refactored it similarly but don't really understand it's significance.

EDIT: I'd rather avoid extensive changes if possible as this PR is relatively easy to test as-is